src/Tools/Source/CompilerGeneratorTools/Source/BoundTreeGenerator/BoundNodeClassWriter.cs

```diff
@@ -1594,6 +1645,11 @@ private static bool SkipInVisitor(Field f)
             return f.SkipInVisitor != null && string.Compare(f.SkipInVisitor, "true", true) == 0;
         }
 
+        private static bool SkipShallowClone(Node n)
+        {
+            return n.SkipShallowClone != null && string.Compare(n.SkipShallowClone, "true", true) == 0;
```

Comment:
The `n.SkipShallowClone != null` check seems unnecessary. (Same comment for similar existing methods above.)